system/settings/settings.xml

```diff
@@ -564,6 +564,14 @@
           </constraints>
           <control type="list" format="string" />
         </setting>
+        <setting id="videoplayer.adjustresolution" type="boolean" label="13549" help="36163">
```

Comment:
please drop the setting. we already have much to many of those. and they are pest when it comes to rebase